src/vs/workbench/contrib/webview/browser/pre/index-no-csp.html

```diff
@@ -536,8 +536,13 @@
 						event.view.scrollTo(0, 0);
 					} else if (node.hash && (node.getAttribute('href') === node.hash || (baseElement && node.href === baseElement.href + node.hash))) {
 						const fragment = node.hash.slice(1);
-						const scrollTarget = event.view.document.getElementById(fragment) ?? event.view.document.getElementById(decodeURIComponent(fragment));
-						scrollTarget?.scrollIntoView();
+						const decodedFragment = decodeURIComponent(fragment);
+						const scrollTarget = event.view.document.getElementById(fragment) ?? event.view.document.getElementById(decodedFragment);
+						if (scrollTarget) {
+							scrollTarget.scrollIntoView();
+						} else if (decodedFragment === 'top') {
```

Comment:
Can you please make this a case insensitive comparison to match the spec?

Reply:
Whoops, good call. 👍